app/models/ontology_version.rb

```diff
@@ -17,6 +17,7 @@ class OntologyVersion < ActiveRecord::Base
   belongs_to :user
   belongs_to :ontology, :counter_cache => :versions_count
   belongs_to :commit
+  has_and_belongs_to_many :provers
```

Comment:
please document the use of provers here. I assume that these are the provers that in principle can be used for proving with the ontology?